OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -22,6 +22,23 @@
 
 namespace OpenRA.Mods.Common.AI
 {
+	class CapturerTarget<TInfoType> where TInfoType : class, ITraitInfoInterface
```

Comment:
I didn't want to imply `Capture` vs `ExternalCapture` or anything else.
I can of course change this.